.circleci/config.yml

```diff
@@ -66,6 +66,9 @@ jobs:
       - store_artifacts:
           path: tests/_output/
 
+      - store_artifacts:
```

Comment:
@kalabro what's the diff between this and `store_test_results` ?

Reply:
@spleshka store_test_results outputs test suite statistics (you will see them here https://circleci.com/build-insights/gh/systemseed/falcon/master) while store_artifacts stores screenshots.